examples/c5g7_transient/rod_translation/surfaces.py

```diff
@@ -0,0 +1,36 @@
+import openmc
```

Comment:
File is identical to `density_change/surfaces.py`. Consider removing and adding a symbolic link instead